src/extension/gdal/src/main/java/org/geoserver/gdal/GDALStatus.java

```diff
@@ -68,9 +68,13 @@ public boolean isEnabled() {
         return Optional.ofNullable("");
     }
 
-    public String getGDALWrapperJarVersion() {
+    public Optional<String> getGDALWrapperJarVersion() {
         if (isAvailable()) {
-            return GeoTools.getVersion(gdal.class).toString();
+            Version v = GeoTools.getVersion(gdal.class);
+            if (v == null) {
+                return Optional.empty();
```

Comment:
I'm not sure when this would return null - gdal.class should be available as its part of the module that this class is in.  This might be an issue with the #getVersion() method or your deployment (i.e. if it's running directly on classes instead of a .jar).  

Looking at the implementation;
https://github.com/geotools/geotools/blob/master/modules/library/metadata/src/main/java/org/geotools/factory/GeoTools.java#L431

it will return null if the version is unavailable, so this should be fine.  

Are you running directly on .class files or does your jar have not have a "abc-version.jar" filename?